src/EFCore.PG/Query/NpgsqlSqlExpressionFactory.cs

```diff
@@ -835,6 +837,21 @@ when left.Type.FullName is "NodaTime.Instant" or "NodaTime.LocalDateTime" or "No
                 break;
             }
 
+            case PgExpressionType.CubeNthCoordinate:
```

Comment:
As an alternative to these, I think you should be able to simply set the returned type mapping (double) directly in the method translator, no? In fact, that's usually better, since we know at the point of translation exactly what type the method returns (it's documented) - the return type isn't sensitive to context and shouldn't get inferred.

Reply:
Just a note that this is still pending.